modules/client-handler/src/main/java/org/apache/ignite/client/handler/ClientHandlerModule.java

```diff
@@ -258,6 +258,15 @@ public InetSocketAddress localAddress() {
         return (InetSocketAddress) ch.localAddress();
     }
 
+    /** Enables request handling. */
+    public void enable() {
+        Channel channel0 = channel;
+
+        assert channel0 != null;
```

Comment:
I suggest to check it on null explicitly and throw exception in case channel is null, like a 
``if (channel0 == null) {
         throw new IgniteInternalException(INTERNAL_ERR, "ClientHandlerModule has not been started"); 
 }``

Reply:
Fixed, thanks.